AdyenCheckout/CheckoutComponentBuilder.swift

```diff
@@ -57,6 +57,19 @@ extension BLIKPaymentMethod {
         } else {
             blikConfiguration = .init()
         }
-        return BLIKComponent(paymentMethod: self, context: AdyenContext.defaultValue(), configuration: blikConfiguration)
+        
+        let context = AdyenContext(
+            apiContext: configuration.apiContext,
+            payment: nil,
+            amount: configuration.amount,
+            analyticsConfiguration: configuration.analyticsConfiguration
+        )
+        var component = BLIKComponent(
+            paymentMethod: self,
+            context: context,
+            configuration: blikConfiguration
+        )
+        component.setBaseCallbacks(from: configuration)
```

Comment:
It is not clear why do we this after we instantiated the component from the configuration, from what I see it could be an implementation details without exposing this here.